packages/appstore/src/components/currency-switcher-card/real/real-account-card.tsx

```diff
@@ -28,6 +31,14 @@ const RealAccountCard = observer(() => {
 
     const get_currency = (IsIconCurrency(currency?.toUpperCase()) && currency) || 'USD';
 
+    const disabledButtonAction = (e: React.MouseEvent) => {
```

Comment:
@sergei-deriv better to rename it to some thing more comprehensive
```suggestion
    const onButtonAction = (e: React.MouseEvent) => {
```

Reply:
@mahdiyeh-deriv Agree, I'll rename it